packages/sit-onyx/src/components/OnyxCheckbox/OnyxCheckbox.ct.tsx

```diff
@@ -2,65 +2,77 @@ import { expect, test } from "../../playwright-axe";
 import OnyxCheckbox from "./OnyxCheckbox.vue";
 
 test("should render unchecked", async ({ mount, makeAxeBuilder }) => {
-  // ARRANGE
-  const component = await mount(
-    <div style="display: grid; width: max-content;">
-      <OnyxCheckbox label="Unchecked" />
-      <OnyxCheckbox label="Hover" />
-      <OnyxCheckbox label="Focus visible" />
-    </div>,
-  );
+  const testCases = [{ name: "default" }, { name: "required", required: true }];
 
-  // ACT
-  const accessibilityScanResults = await makeAxeBuilder().analyze();
+  for (const testCase of testCases) {
```

Comment:
I guess this is the equivalent of test.each in the playwright world: https://playwright.dev/docs/test-parameterize#parameterized-tests 

_if_ you want to change the order like in the example :) but you don't have to of course :)

Reply:
Nice thanks! That looks better in the dev tools :) I also remove redundant test cases for the required indicator.
In don't think we need additional screenshots for unchecked, checked, indeterminate etc. which only include the additional asterisk. I changed it to one screenshot